components/ForgotPassword.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
+
+import Image from "next/image";
+import { useState } from "react";
+
+// Logo Import
+import logo from "../public/Cacart_logo.png";
+
+// Image Import
+import forgotPasswordImage from "../public/forgotPassword.png";
+import dottedShapeImage from "../public/Dotted Shape.png";
+
+export default function ForgotPassword() {
+  const [userData, setUserData] = useState({
+    email: "",
+  });
+  const [emailError, setEmailError] = useState("");
+  const [emailValid, setEmailValid] = useState(false);
+
+  // Value update handler
+  const valueUpdateHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    setUserData((prevData) => ({
```

Comment:
We wouldn't need this approach as we only have `email` field in the `userData` object. 
`serUserData({ email: value })` would be sufficient enough

Reply:
DONE